trajectory_tracker/test/include/trajectory_tracker_test.h

```diff
@@ -183,37 +186,38 @@ class TrajectoryTrackerTest : public ::testing::Test
   }
   void waitUntilStart(const std::function<void()> func = nullptr)
   {
-    ros::Rate rate(50);
-    const auto start = ros::WallTime::now();
-    while (ros::ok())
+    rclcpp::Rate rate(50);
+    rclcpp::Clock clock;
+    const auto start = clock.now();
+    while (rclcpp::ok())
     {
       if (func)
         func();
 
       publishTransform();
       rate.sleep();
-      ros::spinOnce();
+      rclcpp::spin_some(node_);
       if (status_ &&
-          status_->status == trajectory_tracker_msgs::TrajectoryTrackerStatus::FOLLOWING)
+          status_->status == trajectory_tracker_msgs::msg::TrajectoryTrackerStatus::FOLLOWING)
         break;
-      ASSERT_LT(ros::WallTime::now(), start + ros::WallDuration(10.0))
+      ASSERT_LT(clock.now(), start + rclcpp::Duration::from_seconds(10.0))
           << "trajectory_tracker status timeout, status: "
           << (status_ ? std::to_string(static_cast<int>(status_->status)) : "none");
     }
-    initial_cmd_vel_time_ = ros::Time::now();
+    initial_cmd_vel_time_ = clock.now();
```

Comment:
use clock bundled with the node
```suggestion
    initial_cmd_vel_time_ = node_->now();
```